### DISCUSSION TITLE
Communication Channels

### DISCUSSION THREAD

**Discussion by binnes** [okd-project/okd#1207] [General]
We currently have the [Kubernetes slack channel](https://kubernetes.slack.com/archives/C6AD6JM17) and the discussions in this git repo for community memebers to discuss OKD.  There is the Facebook group, the Google group and a twitter account.

A Matrix channel has also been setup, but is not being used.

What is the community view on communication channels?  How many should we have and where should they be?

**Comment by vrutkovs** [comment]
It would be valuable to have various channels - especially if we could bridge them all in one (like Matrix can). If we can't bridge them together lets group them in "official" (Slack + mailing list) and "additional"?
